application/services/stripe_client.py

```diff
@@ -1,19 +1,30 @@
-import stripe
+import stripe, time, inspect, distutils, sys
 from datetime import datetime, timedelta, date
-import time
 from flask import current_app
+from flask.ext.login import current_user
+
+try:
+    from application.util import error, logger, constants
+except ImportError:
+    import sys, os
+    sys.path.append(os.path.join(os.path.dirname(__file__), '..', 'util'))
```

Comment:
I don't think we can reference local file inside app engine. Instead, we should import everything from top level?